src/components/ModalGlobal.vue

```diff
@@ -0,0 +1,56 @@
+<template>
+  <Modal
+    style="z-index:1000"
+    :okText="okText"
+    :cancelText="cancelText"
+    :titleText="titleText"
+    :bodyText="bodyText"
+    :showCancel="showCancel"
+    @ok="okCallback"
+    @close="cancelCallback"
+    ref="globalModal"
+  ></Modal>
+</template>
+
+<script>
+import Modal from './Modal.vue';
+  export default {
+    name: 'ModalGlobal',
+    props: [],
+    components:{
+        Modal
+    },
+    data: function(){
+        return {
+            okText: undefined,
+            cancelText: undefined,
+            titleText: undefined,
+            bodyText: undefined,
```

Comment:
```suggestion
            okText: null,
            cancelText: null,
            titleText: null,
            bodyText: null,
```

Reply:
This is actually intended, as vue would consider undefined to, you know, undefined. If I pass null it will indeed clear all the display text to null, which is not what I wanted. Is it ok to set vars be undefined?